.github/workflows/validate-pr.yml

```diff
@@ -21,6 +21,13 @@ jobs:
           if ["${{ secrets.SALESFORCE_USERNAME }}" == ""]; then echo "::error title=Missing input value::Repository secret SALESFORCE_USERNAME has not been set. See the Readme for more details." && exit 1; fi;
           if ["${{ secrets.SALESFORCE_PASSWORD }}" == ""]; then echo "::error title=Missing input value::Repository secret SALESFORCE_PASSWORD has not been set. See the Readme for more details." && exit 1; fi;
 
+      # Check out the merge result of the PR
+      - name: Check out merge result
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.merge_commit_sha }}

```

Comment:
Is this definitely the correct variable? I was expecting `github.event.pull_request.merge_commit_sha`

Reply:
It definitely works, it yields a full commit hash.

But I think maybe we should just remove the `ref` input altogether because I think when triggered by a PR, that's the default behavior?

Reply:
Or, hm... actually maybe it was the `checkout` action that printed it.